bot/cogs/watchchannels/bigbrother.py

```diff
@@ -52,6 +52,16 @@ def __init__(self, bot: Bot) -> None:
         A `reason` for adding the user to Big Brother is required and will be displayed
         in the header when relaying messages of this user to the watchchannel.
         """
+        await self.apply_watch(ctx, user, reason)
+
+    @bigbrother_group.command(name='unwatch', aliases=('uw',))
+    @with_role(*MODERATION_ROLES)
+    async def unwatch_command(self, ctx: Context, user: FetchedMember, *, reason: str) -> None:
+        """Stop relaying messages by the given `user`."""
+        await self.apply_unwatch(ctx, user, reason)
+
+    async def apply_watch(self, ctx: Context, user: FetchedMember, reason: str) -> None:
+        """Handles adding a user to the watch list."""
```

Comment:
This doesn't do the function justice
```suggestion
        """
        Add `user` to watched users and apply a watch infraction with `reason`.

        A message indicating the result of the operation is sent to `ctx`.
        The message will include `user`'s previous watch infraction history, if it exists.
        """
```